Libs/Gtk3/Classes/TextView.cs

```diff
@@ -4,32 +4,58 @@ namespace Gtk
 {
     public partial class TextView
     {
-        public TextView() {}

```

Comment:
Copy/pasted from above:

> **firox263:** Empty constructors should be fine since they propagate via the `GObject(params ConstructParameter[])` constructor (with no parameters provided, it is effectively empty). By design, everything except the IntPtr constructor goes via `g_object_new_with_properties` in the GObject empty base constructor, so just about anything should work for constructor signatures.

Reply:
Ah you are right, we should restore it. But it was very unobvious what is happening. I think it is much clearer if we define it like this:

`public TextView() : base(Array.Empty<ConstructParameter>()) { }`

Reply:
Would Array.Empty allocate an object here? As that might be slightly less efficient than simply passing nothing. 

We could/should explicitly chain it using either `this` or `base` though to make it clear what's happening.

Reply:
@firox263 AFAIK `Array.Empty` allocates nothing and just return a reference to a static generic class with a static field holding the generic array (https://source.dot.net/#System.Private.CoreLib/Array.cs,702), so I think it should be fine to use it.

Reply:
Fixed.